src/Core/AdminConsole/Services/Implementations/OrganizationService.cs

```diff
@@ -358,6 +358,11 @@ private async Task<string> AdjustSeatsAsync(Organization organization, int seatA
             }
         }
 
+        if (organization.UseSecretsManager && organization.Seats + seatAdjustment < organization.SmSeats)
+        {
+            throw new BadRequestException("You cannot have more Password Manager seats than Secrets Manager seats.");
```

Comment:
Just spotted this. I think this logic is correct, but the error message is around the wrong way. You cannot have more SM seats than PM seats. (Every SM user must also be a PM user.)